### ISSUE TITLE
Rat in a maze problem - Backtracking 

### ISSUE THREAD

**Issue by dsrao711** [dsrao711/DSA-Together-HacktoberFest#3]
Link to the problem: https://practice.geeksforgeeks.org/problems/rat-in-a-maze-problem/1

**Comment by saaiiravi** [comment]
Can you assign this to me?

**Comment by dsrao711** [comment]
Ya sure. Go ahead @saaiiravi

**Comment by dsrao711** [comment]
@saaiiravi  Are you still working on this ?

**Comment by tomvapon** [comment]
Hi! I would like to work on this issue if it is possible.

**Comment by dsrao711** [comment]
Ok @saaiiravi

**Comment by dsrao711** [comment]
@tomvapon, issues are assigned on First come first serve basis, You can work on other issues

**Comment by tomvapon** [comment]
> @tomvapon, issues are assigned on First come first serve basis, You can work on other issues

Yes @dsrao711, I know, It was quite an interesting issue to me, and I was asking in order to know if @saaiiravi was not available to do it.

Btw, I have coded a little version, let me know if you want a hint, I can help here =D.

Good coding! :)
